### PR TITLE
Update Food.pm (Handling added sugars)

### PR DESCRIPTION
**Pull request by teolemon**
Handling added sugars

### CODE REVIEW

File: lib/ProductOpener/Food.pm [hunk 1]
```diff
@@ -1444,6 +1445,10 @@ sub mmoll_to_unit {
 		zh_HK => "糖",
 		zh_TW => "糖",
 	},
+	added-sugars => {
```
Comment (stephanegigandet) [inline lib/ProductOpener/Food.pm:1448]:
```suggestion
	"added-sugars" => {
```

il faut mettre des guillements autour de l'identifiant (sauf quand il n'y a pas de tiret auquel cas ça marche sans)

File: lib/ProductOpener/Food.pm [hunk 2]
```diff
@@ -4241,7 +4246,7 @@ sub special_process_product($) {
 				or has_tag($product_ref, "ingredients", "sirop-de-fructose") or has_tag($product_ref, "ingredients", "saccharose")
 				or has_tag($product_ref, "ingredients", "sirop-de-fructose-glucose") or has_tag($product_ref, "ingredients", "sirop-de-glucose-fructose-de-ble-et-ou-de-mais")
 				or has_tag($product_ref, "ingredients", "sugar") or has_tag($product_ref, "ingredients", "sugars")
-
+				or has_tag($product_ref, "ingredients", "added sugar") or has_tag($product_ref, "ingredients", "added sugars")
```
Comment (stephanegigandet) [inline lib/ProductOpener/Food.pm:4249]: this is for ingredients, not nutrients, so "added sugars" should not be added here.

Comment (teolemon) [inline lib/ProductOpener/Food.pm:4249]:
```suggestion
```

**Comment by teolemon** [comment]
fixed @stephanegigandet

**Pull request merged**